schema/regions/sewer.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "regions_sewer",
+  "additionalProperties": false,
+  "required": [
+    "week_unix",
+    "week_start_unix",
+    "week_end_unix",
+    "vrcode",
+    "average",
+    "total_installation_count",
+    "date_of_insertion_unix"
+  ],
+  "properties": {
+    "week_unix": {
+      "type": "integer"
+    },
+    "week_start_unix": {
+      "type": "integer"
+    },
+    "week_end_unix": {
+      "type": "integer"
+    },
+    "vrcode": {
+      "type": "string",
+      "pattern": "^VR[0-9]+$"
```

```suggestion
      "equalsRootProperty": "code"
```
This checks if the vrcode is the same as the 'code' property in the root file.